common/spotbugsExclude.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+  <Match>
+    <Class name="~.*\.generated.*"/>
```

Comment:
Spotbugs is probably redundant with the use of sonarqube, but catalog is still using both too. Anyway catalog is using spotbugs but doesn't need to specify a filter like this; I think the generated code is annotated so it isn't scanned by spotbugs. Does the build fail if you remove this filter?